File: c/tests/exists_test.c
// Generated existence test: the SDK constructs in test mode.

#include "ctest.h"

int main(void) {
  SolardemoSDK* testsdk = test_sdk(NULL, NULL);
  CHECK(testsdk != NULL, "test_sdk returns a client");
  CHECK_STR_EQ(testsdk->mode, "test", "test_sdk mode is test");
  TEST_SUMMARY("exists");
}

File: rust/tests/exists_test.rs
// Generated existence test: the SDK constructs in test mode.

use solardemo_sdk::{test_sdk, Value};

#[test]
fn exists_test_mode() {
    let testsdk = test_sdk(Value::Noval, Value::Noval);
    assert_eq!(*testsdk.mode.borrow(), "test");
}
